### ISSUE TITLE
[BUG] Where you left off... is incomplete

### ISSUE THREAD

**Issue by mrburns-42** [LeoKlaus/plappa#79]
**Describe the bug**
When I set up ABS, I took over the audiobook position of all the audiobooks that I had started in recent years. This means that I have around 55 audiobooks that I have started but not yet finished. In Plappa, only three of the 55 audiobooks are displayed under "Where you left off...". Clicking on the arrow to see the total overview of audiobooks started also only shows me three audiobooks.

When I have finished one of the three audiobooks, only two are still displayed. If I start three new audiobooks, these are displayed correctly under "Where you left off...".

I have reinstalled the app several times to test it and it always shows me the last three audiobooks that have not yet been finished and that I listened to last.

**To Reproduce**
1. Install plappa
2. Complete the initial setup
3. Click on the arrow next to "Where you left off..."
4. See only three audiobooks

**Expected behavior**
See my 55 unfinished audiobooks.

**Environment(please complete the following information):**
 - Device: iPhone 15 Pro
 - OS: iOS 17.4,1
 - Version 1.1.2 - 1.1.4
 - Server Version ABS 2.9.0

**Comment by LeoKlaus** [comment]
plappa only shows audiobooks that have seen progress in the past 14 days in "Where you left off...". Unless those books missing in "Where you left off..." are also missing their playback states, this is not a bug, it's a feature ;)

I added this limit back when the UI was working very differently, but I don't see why I should keep it in place now. I've already implemented this change and have submitted the update to TestFlight (still pending review).
